dbms/src/Functions/FunctionsFindCluster.h

```diff
@@ -270,35 +142,129 @@ class FunctionFindClusterIndex : public IFunction
         return ClusterOperation::FindClusterIndex;
     }
 
+    virtual void executeImplTyped(
```

Comment:
Function should not be virtual, unless you are going to override it and then to use the object in polymorphic way.